projects/valtimo/config/src/lib/config.service.ts

```diff
@@ -51,6 +52,10 @@ export class ConfigService {
         ...config.valtimoApi,
         endpointUri: this.formatUrlTrailingSlash(config.valtimoApi.endpointUri, true),
       },
+      translationResources: config.translationResources?.map<ITranslationResource>(
```

Comment:
Since translationResources is not always included in the environment file (optional property), I think it's better if you conditionally add this key to the environment object instead of checking if it's truthy after already putting in the key. See the first answer here for some syntax to make this possible: https://stackoverflow.com/questions/11704267/in-javascript-how-to-conditionally-add-a-member-to-an-object